great_expectations/core/expectation_validation_result.py

```diff
@@ -182,6 +188,35 @@ def __str__(self) -> str:
         """
         return json.dumps(self.to_json_dict(), indent=2)
 
+    def render(self) -> None:
+        """Renders content using the:
+        - atomic prescriptive renderer for the expectation configuration associated with this
+          ExpectationValidationResult to self.expectation_config.rendered_content
+        - atomic diagnostic renderer for the expectation configuration associated with this
+          ExpectationValidationResult to self.rendered_content.
+        """
+        inline_renderer_config: Dict[str, Union[str, ExpectationValidationResult]] = {
+            "class_name": "InlineRenderer",
+            "render_object": self,
+        }
+        module_name: str = "great_expectations.render.renderer.inline_renderer"
+        inline_renderer = instantiate_class_from_config(
+            config=inline_renderer_config,
+            runtime_environment={},
+            config_defaults={"module_name": module_name},
+        )
```

Comment:
Could we instantiate the object with the constructor instead using `instantiate_class_from_config`? Since we know the type, we don't necessarily need the magic method right?

Reply:
1) This follows the existing pattern for renderers, but if it is no longer preferred we can use the class. 2) This will result in a circular import that I will need to figure out how to address in the renderer.

Reply:
> Would this still be an issue if we imported within the function scope?
You mean like this?
```
def render(self) -> None:
        from great_expectations.render.renderer.inline_renderer import InlineRenderer

        inline_renderer = InlineRenderer(render_object=self)

        (
            self.expectation_config.rendered_content,
            self.rendered_content,
        ) = inline_renderer.render()
```
That works, but I'm not sure that I've seen it anywhere else in the codebase. The options seem to be:
1. Do it with `instantiate_class_from_config` like it is now.
2. Import `InlineRenderer` at the top of the module and remove `ExpectationConfiguration` and `ExpectationValidationResult` objects from the renderer - inferring them from their attributes. And then maybe update the other renderers to follow this pattern.
3. Import `InlineRenderer` in the `render` method, and then maybe update the other renderers to follow this pattern.